core/core-platform-bukkit/src/main/java/net/lax1dude/eaglercraft/backend/server/bukkit/BukkitUnsafe.java

```diff
@@ -544,8 +544,17 @@ public static boolean isEnableNativeTransport(Server server) {
 				Method getBoolean = propertyManager.getClass().getSuperclass().getDeclaredMethod("getBoolean", String.class, boolean.class);
 				getBoolean.setAccessible(true);
 				return (Boolean) getBoolean.invoke(propertyManager, "use-native-transport", true);
-			} catch (Exception e1) {
-				throw Util.propagateReflectThrowable(e1);
+			} catch (ReflectiveOperationException e1) {
+				try {
+					Object dedicatedPlayerList = server.getClass().getMethod("getHandle").invoke(server);
+					Object dedicatedServer = dedicatedPlayerList.getClass().getMethod("b").invoke(dedicatedPlayerList);
+					Object propertyManager = dedicatedServer.getClass().getMethod("a").invoke(dedicatedServer);
+					Method getBoolean = propertyManager.getClass().getSuperclass().getDeclaredMethod("b", String.class, boolean.class);
```

Comment:
Can we combine this try/catch with its parent and match obfuscated methods through iterating each method and comparing their signatures like the other obfuscation helpers do?